<launch>
  <env name="ROSCONSOLE_FORMAT" value="[${node}][${time:%H:%M:%S%F}][${severity}]:${message}"/>
  <arg name="map_size_x" value="10" />
  <arg name="map_size_y" value="10" />
  <arg name="map_size_z" value="3" />
  <arg name="obs_num" default="6" />
  <!-- Number of obstacles -->
  <arg name="num_robots" value="1" />
  <arg name="rviz" default="True" />

  <include file="$(find planner)/launch/simulator/simulator_fake_real.launch">
    <arg name="map_mode" value="3" />
    <arg name="map_size_x" value="$(arg map_size_x)" />
    <arg name="map_size_y" value="$(arg map_size_y)" />
    <arg name="map_size_z" value="$(arg map_size_z)" />
    <arg name="obs_num" value="$(arg obs_num)" />
  </include>

  <group ns="uav0">
    <include file="$(find planner)/launch/simulator/drone_fake_perception.xml">
      <arg name="drone_id" value="0" />
      <arg name="num_robots" value="$(arg num_robots)" />
      <arg name="map_size_x" value="$(arg map_size_x)" />
      <arg name="map_size_y" value="$(arg map_size_y)" />
      <arg name="map_size_z" value="$(arg map_size_z)" />
      <arg name="init_x" value="0.363" />
      <arg name="init_y" value="0.600" />
      <arg name="init_z" value="1.000" />
      <arg name="goal_x" value="-2.22" />
      <arg name="goal_y" value="0" />
      <arg name="goal_z" value="1.000" />
    </include>
  </group>

<group if="$(arg rviz)">
  <node pkg="rviz" name="rviz" type="rviz" args="-d $(find planner)/rviz/demo.rviz" />
</group>

</launch>
